src/main/java/io/peacemakr/crypto/ICrypto.java

```diff
@@ -62,7 +62,25 @@
   byte[] decrypt(byte[] cipherText) throws PeacemakrException;
 
   /**
-   * For visibility or debugging purposes, returns a string whihc identifies which
+   * Sing the message.
+   * <p>
+   * @param plainText clear text.
+   * @return A signedBlob upon success.
+   * @throws PeacemakrException if an exception occurs during signing attempt.
+   */
+  byte[] signOnly(byte[] plainText) throws PeacemakrException;
+
+  /**
+   * Verify the signed blob. VerifyOnly only works on blob returned by the result of SignOnly().
+   * <p>
+   * @param signedBlob packaged blob containing unencrypted data.
+   * @return message in the clear upon success.
```

Comment:
We mean, "... upon successful verification" maybe? ... and.... what happens when verification is not successful?

Reply:
This is an example of what c lib + JNI will produce if verify fails:
```
asymmetric_verify: 197 - error:00000000:lib(0):func(0):reason(0)
asymmetric_verify: 198 - DigestVerifyFinal failed
Verification of the message failed
``` 

the returned byte[] will be null, hence checking for null before returning. If C.verify() returns null, javaSDK throws an exception